src/Agent.Worker/TestResults/ResultsCommandExtension.cs

```diff
@@ -289,8 +288,10 @@ private async Task PublishTestRunDataAsync(String teamProject, TestRunContext te
             }
 
             await PublishEventsAsync(connection);
+            TestResultsHttpClient tcmClient = connection.GetClient<TestResultsHttpClient>();
+            await tcmClient.UpdateCodeCoverageSummaryAsync(_executionContext.Variables.System_TeamProjectId.ToString(), _executionContext.Variables.Build_BuildId.GetValueOrDefault());
```

Comment:
@anshii03 What's the roll out strategy ? isn't this should be behind a FF. 
If not the calls will be happening  for all the pipelines & run complete/build complete event will also be doing the same thing. Will that cause any issue?

Reply:
@navin22 Putting behind feature flag is preferred just in case we see some unexpected issue/regression. Feature flag should be for safe roll out of the changes. 
i.e. enable FF for few accounts initially, wait for few days/weeks before enabling for all scale units.

Reply:
I understand this part, my reason for having a FF is for 
Currently there’s an event handler which does the same thing of updating the coverage summary once the run is complete. Is that still active ? 
If Yes, this call from the pipeline will be a redundant & do we know the consequences because now there will be two flows which will be trying to update coverage summary?

Reply:
@navin22 Currently we have PATCH Api call in Vstest task. This is the first call we are adding in dotnet cli task.

Reply:
I think I am not getting the full picture here, there's a code in tcm service which actually triggers on test run complete and calls codecoverage APIs. Is that and `UpdateCodeCoverageSummaryAsync` is same?

Reply:
@navin22 Can you please share the code you are referring to ?